src/generator/AutoRest.Ruby.Azure/Templates/AzureMethodTemplate.cshtml

```diff
@@ -92,6 +97,10 @@ def @(Model.Name)_async(@(Model.MethodParameterDeclaration))
     {
       @:@(Model.DeserializePollingResponse("parsed_response", Model.ReturnType.Body))
     }
+    else
+    {
+      @:parsed_response

```

Comment:
@sarangan12 what's in parsed_response? what ends up in "deserialize_method" and being passed to "get_long_running_operation_results"?